src/components/Layout/index.tsx

```diff
@@ -8,9 +8,23 @@ interface Props {
   title?: string;
 }
 
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace JSX {
+    interface IntrinsicElements {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      'dc-header': any;
+    }
+  }
+}
+
 const Layout: React.FC<Props> = ({ children, title, description }) => {
   return (
     <>
+      <dc-header
+        host="http://debtcollective.org"
```

Comment:
This part suck a bit right now as this value doesn't adjust for local development.
_I am assuming here that this page is going to live under http://debtcollective.org_ with that being said, we may need to challenge some of the links described below